### ISSUE TITLE
[bug] Localstack docker image started but not visible in docker

### ISSUE THREAD

**Issue by MarcelStranak** [localstack/localstack-docker-extension#29]
Using this version:  `pive01/localstack-docker-extension:0.3.2`

Used config:
LOCALSTACK_API_KEY=XXX, DEBUG=1, LOCALSTACK_VOLUME_DIR=/C/temp/localstack

When hitting the start button, I received all the INFO, DEBUG messages as toast notifications:
![image](https://user-images.githubusercontent.com/97876429/221179772-8bf04989-cefc-4b91-9468-06ee75f9843e.png)
![image](https://user-images.githubusercontent.com/97876429/221179881-050adfb9-ece6-47ad-a307-252f9f29e6d5.png)

I can see that the docker image is started:
![image](https://user-images.githubusercontent.com/97876429/221180248-d593db55-d99e-4a60-bb9b-146bff8829dd.png)

Running `localstack status` to get info if localstack is started:

```
localstack status                                                                                                                            

┌─────────────────┬───────────────────────────────────────────────────────┐
│ Runtime version │ 1.4.1.dev                                             │
│ Docker image    │ tag: latest, id: 48775529339f, 📆 2023-02-23T12:47:41 │
│ Runtime status  │ ✖ stopped                                             │
└─────────────────┴───────────────────────────────────────────────────────┘
```

Testing out if it works anyway.
```
awslocal docdb create-db-cluster --db-cluster-identifier fb231 --engine docdb  
{
    "DBCluster": {
        "DBClusterIdentifier": "fb231",
        "DBClusterParameterGroup": "default.docdb",
        "Status": "available",
        "Endpoint": "localhost:38397",
        "MultiAZ": false,
        "Engine": "docdb",
        "Port": 38397,
        "MasterUsername": "test",
        "DBClusterMembers": [],
        "StorageEncrypted": false,
        "DBClusterArn": "arn:aws:rds:us-east-1:000000000000:cluster:fb231"
    }
}
```

If I check Docker Desktop Containers page if the container is running it's not mentioned there. Only the docdb image is visible.

![image](https://user-images.githubusercontent.com/97876429/221180810-37104802-0f52-4452-a4c2-5dbfd078a6a9.png)

Logs:
```

LocalStack version: 1.4.1.dev
LocalStack Docker container id: 1a1f007cd964
LocalStack build date: 2023-02-23
LocalStack build git hash: 4be57540

Starting DNS servers (tcp/udp port 53 on 0.0.0.0)...
Ready.

2023-02-24T12:33:29.996  INFO --- [  MainThread] l.extensions.platform      : loaded 0 extensions
2023-02-24T12:33:30.003  INFO --- [  MainThread] botocore.credentials       : Found credentials in environment variables.
2023-02-24T12:33:30.042  WARN --- [-functhread5] hypercorn.error            : ASGI Framework Lifespan error, continuing without Lifespan support
2023-02-24T12:33:30.042  WARN --- [-functhread5] hypercorn.error            : ASGI Framework Lifespan error, continuing without Lifespan support
2023-02-24T12:33:30.049  INFO --- [-functhread5] hypercorn.error            : Running on https://0.0.0.0:4566 (CTRL + C to quit)
2023-02-24T12:33:30.049  INFO --- [-functhread5] hypercorn.error            : Running on https://0.0.0.0:4566 (CTRL + C to quit)
2023-02-24T12:33:30.070  WARN --- [-functhread6] hypercorn.error            : ASGI Framework Lifespan error, continuing without Lifespan support
2023-02-24T12:33:30.070  WARN --- [-functhread6] hypercorn.error            : ASGI Framework Lifespan error, continuing without Lifespan support
2023-02-24T12:33:30.070  INFO --- [-functhread6] hypercorn.error            : Running on https://0.0.0.0:443 (CTRL + C to quit)
2023-02-24T12:33:30.070  INFO --- [-functhread6] hypercorn.error            : Running on https://0.0.0.0:443 (CTRL + C to quit)
2023-02-24T12:35:27.076  INFO --- [   asgi_gw_0] botocore.credentials       : Found credentials in environment variables.
2023-02-24T12:35:27.278  INFO --- [functhread27] l.services.docdb.docdb_api : Starting Document DB instance on port 38397
2023-02-24T12:35:29.718  INFO --- [   asgi_gw_0] localstack.request.aws     : AWS rds.CreateDBCluster => 200
```

**Comment by Pive01** [comment]
Thanks, @MarcelStranak ..... I think you don't see it because you have an option disabled ...can you go to Settings >Extensions > Show Docker Extensions system container and give it another try? (Should add this in the doc or somewhere)....Also, the DEBUG=1 toasts need to be addressed 😅

**Comment by MarcelStranak** [comment]
Thanks. I was able to activate it and see the containers.
![image](https://user-images.githubusercontent.com/97876429/221766174-84801d6c-ab3c-4a78-9e71-209e4150a9f7.png)
![image](https://user-images.githubusercontent.com/97876429/221766210-1de69490-78c5-4a65-a934-e0c713ff337c.png)

I noticed I have this log message in there:
![image](https://user-images.githubusercontent.com/97876429/221766299-f47ad4ed-2960-4715-9172-acf11a133207.png)

Anyhow, I can confirm that the image is working. It still seems to be a problem if you are also using LocalStack CLI and try to use for example `localstack status` and it does not work.
